data-generation/src/main/scala/UpdateDistributions.scala

```diff
@@ -129,9 +138,77 @@ object UpdateDistributions {
     distribution match {
       case DistributionType.LogNormalType(mu, sigma) => LogNormal(mu, sigma).draw().toInt
       case DistributionType.GaussianType(mu, sigma) => Gaussian(mu, sigma).draw().toInt
+      case DistributionType.UniformType(low, high) => Uniform(low, high).draw().toInt
     }
   }
 
+  private def getSortedUniformDistribution(low:Double, high:Double, numSamples:Int):List[Long] =
+    Uniform(low,high).sample(numSamples).map(_.toLong).toList.sorted
+
+  def generateTSVs(graph: Graph[Int, IntervalAndUpdateCount]): RDD[LogTSV] = {
```

Comment:
Føler disse funksjonene som genererer logs burde være definert i LogTSV, og ikke her

Reply:
Og kan vi kalle det log istedenfor "tab separated values", typ
```suggestion
  def generateLogs(graph: Graph[Int, IntervalAndUpdateCount]): RDD[LogTSV] = {
```

Reply:
Den er veldig spesifikk for denne type data. Med manglende updates og kun interval for edges. 
Og ja, kan begynne å kalle det logs

Reply:
Forsåvidt enig i at denne er det, så ikke så mye de hver for seg. Men de andre er mindre spesifikke for denne, legger til kommentar